src/highlights-list/index.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Highlights list block
+ * Frontend PHP code
+ *
+ * Uses WordPress' dynamic block method
+ * https://developer.wordpress.org/block-editor/tutorials/block-tutorial/creating-dynamic-blocks/
+ *
+ * @package mojblocks
+ *
+ */
+
+function render_callback_highlights_list_block($attributes, $content)
+{
+
+    // Parse attributes found in index.js
+    $attribute_list_title = $attributes['listTitle'] ?? '';
+    $attribute_list_items = $attributes['listItems'] ?? '';
+    $attribute_list_className = $attributes['listClassName'] ?? '';
+
+    // Turn on buffering so we can collect all the html markup below and load it via the return
+    // This is an alternative method to using sprintf(). By using buffering you can write your
+    // code below as you would in any other PHP file rather then having to use the sprintf() syntax
+    ob_start();
+
+    ?>
+
+    <div class="<?php _e(esc_html($attribute_list_className)) ; ?> mojblocks-highlights-list">
+        <div class="govuk-width-container">
+            <div class="govuk-grid-row">
+                <div class="mojblocks-highlights-list__heading-container">
+                    <h2 class="mojblocks-highlights-list__heading">
+                                 <span role="text">
+                                    <span class="mojblocks-highlights-list__heading-text">
+                                         <?php _e(esc_html($attribute_list_title)); ?>
+                                    </span>
+                                  </span>
+                    </h2>
+                </div>
+                <div class="mojblocks-highlights-list__content">
+                    <ul>
+                        <?php _e($attribute_list_items); ?>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <?php
+
+    // Get all the html/content that has been captured in the buffer and output via return
+    $output = ob_get_contents();
+    ob_end_clean();
```

Comment:
I saw the other PR had `$output = html_entity_decode($output);` in it - does this need that too?

Reply:
They are not lkely to put other markup in here but its worth having it in there just in case. Good spot.

Reply:
Added

Reply:
:D